test/contrib/test_autoguide.py

```diff
@@ -39,6 +39,7 @@ def body_fn(i, val):
 
```

Comment:
Could you test this on `beta_bernoulli`? I think the diag normal guide gives a somewhat biased estimate for `beta`, and I'm curious to see if this does better.

Reply:
The IAF guide only works with latent_size > 1. I'll see if I can remove that restriction. If not, I'll add another similar test. :)

Reply:
I see. You could also just increase the dimension in that test (for latent and observed data).

Reply:
Looking like bias issue still happens with IAF. I'll try to add multiple IATransform to see if it helps.

Reply:
I haven't looked into this in detail, and that was just my hypothesis. It is also possible that the data size is small. If NUTS gives answers close to the true value, then we know it is a bias issue.

Reply:
It turns out that we should fix the values of true coefs. :D

Reply:
I completely missed that the data has only 10 elements. Makes sense!